From 8c497bc794e1e6a3ed188a548da771d768cef8f1 Mon Sep 17 00:00:00 2001
From: Ken Gaillot <kgaillot@redhat.com>
Date: Wed, 26 Oct 2016 11:18:17 -0500
Subject: [PATCH] Fix: lrmd: undo unnecessary LRMD protocol version change

The change breaks rolling upgrades in a cluster with Pacemaker Remote nodes,
and was never necessary. This introduces a divergence from upstream that
will need to be reconciled in the future.
---
 include/crm/lrmd.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/crm/lrmd.h b/include/crm/lrmd.h
index 446b39c..a099315 100644
--- a/include/crm/lrmd.h
+++ b/include/crm/lrmd.h
@@ -35,7 +35,7 @@ typedef struct lrmd_key_value_s {
     struct lrmd_key_value_s *next;
 } lrmd_key_value_t;
 
-#define LRMD_PROTOCOL_VERSION "1.1"
+#define LRMD_PROTOCOL_VERSION "1.0"
 
 /* *INDENT-OFF* */
 #define DEFAULT_REMOTE_KEY_LOCATION "/etc/pacemaker/authkey"
-- 
1.8.3.1

